enhancements/network/20190903-SRIOV-GA.md

```diff
@@ -12,19 +13,19 @@ reviewers:
 approvers:
   - "@derekwaynecarr"
 creation-date: 2019-09-03
-last-updated: 2019-09-03
-status: implementable
+last-updated: 2020-08-17
+status: provisional
```

Comment:
Did you mean to change this from `implementable` back to `provisional` ?  Can you explain the reason?  Thanks!

Reply:
my bad, it was a mistake, shall be `implemented`.
updated